src/types/interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface User {
```

Comment:
Why are we using interface instead of type? Interface to me is used typically for abstracting behaviours.